### PR TITLE
Implemented multivariate normal density and test.

### PR DESCRIPTION
**Pull request by cschoeller**

### CODE REVIEW

File: bark/commons/tests/distribution_tests.cc [hunk 1]
```diff
@@ -88,15 +90,14 @@ TEST(distribution_test, uniform_dist_1d) {
...
+  //const double upper_bound = 10.0;
+  params_ptr1->SetListListFloat(
       "Covariance", {{1.0, 0.2, 0.1}, {0.2, 3.0, -0.5}, {0.1, -0.5, 0.125553}});
-  params_ptr->SetListFloat("Mean", {1.2, 12.0, 0.1234f});
-  params_ptr->SetInt("RandomSeed", 1000.0);
-
-  auto dist_multivariate = bark::commons::MultivariateDistribution(params_ptr);
+  params_ptr1->SetListFloat("Mean", {1.2, 12.0, 0.1234f});
+  params_ptr1->SetInt("RandomSeed", 1000.0);
+  auto dist_multivariate = bark::commons::MultivariateDistribution(params_ptr1);
```
Comment (patrickhart) [inline bark/commons/tests/distribution_tests.cc:100]: this distribution never seems to be used? Could omit it in the test

**Review by patrickhart — APPROVED** [review]
Looks good to me!

File: bark/commons/tests/distribution_tests.cc (hunk 1 above)
Comment (cschoeller) [reply bark/commons/tests/distribution_tests.cc:100]: Did you expand all code? It is used in line 105 for example.

Comment (patrickhart) [reply bark/commons/tests/distribution_tests.cc:100]: just saw the changes

**Review by juloberno — APPROVED** [review]
Looks good

**Pull request merged**